ethereum/contracts/foundry.toml

```diff
@@ -4,8 +4,7 @@
 # Don't commit changes to this line which point to /nix/store/...
 # These are part of the `nix develop` shell setup.
 # This must be: 0.8.19
-solc = "0.8.19"
-src = 'src'
+solc = "/nix/store/rvhqi1j75gnf0md4x9bvy9wvyy9k2ax8-solc-default/bin/solc"
```

Comment:
This should not be pushed.

Reply:
Oups. Don't know what happened there. It's fixed now.

Reply:
Happens to all of us all the time :)